src/Commands/AdminCommands/ChatsCommand.php

```diff
@@ -58,6 +58,7 @@ public function execute()
         $results = DB::selectChats(
```

Comment:
An option could be to just pop them into the DB class:
```php
const SINGLE_CHAT = 1;
const GROUP_CHAT = 2;
const SUPERGROUP_CHAT = 4;
const CHANNEL = 8;
```
But this may seem a bit confusing 😕 

Probably makes sense to do the same here like with other parameters, passing the values as an array (as described further down from my link above: https://stackoverflow.com/a/29089798):
```php
$results = DB::selectChats([
    'groups'       => true,
    'supergroups'  => true,
    'channels'     => true,
    'single_chats' => true, // or 'user_chats', or just 'single'?
    'from'         => 'yyyy-mm-dd hh:mm:ss', // or better, use DateTimeImmutable objects
    'to'           => 'yyyy-mm-dd hh:mm:ss',
]);
```